src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -154,10 +155,11 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             }
 
             var manifestFileProperties = new Dictionary<string, string>();
+            var nodeCommandManifestFileProperties = new Dictionary<string, string>();
 
             // Write the platform name and version to the manifest file
             manifestFileProperties[ManifestFilePropertyKeys.NodeVersion] = nodePlatformDetectorResult.PlatformVersion;
-
+            nodeCommandManifestFileProperties["PlatformWithVersion"] = "nodejs " + nodePlatformDetectorResult.PlatformVersion;
```

Comment:
No need to put it there because the manifest file already included version information.

Reply:
we need this in this as well, this was a requirement